shell/server/core.js

```diff
@@ -137,6 +137,31 @@ class SandstormCoreImpl {
       logActivity(this.grainId, null, event);
     });
   }
+
+  reportGrainSize(bytes) {
+    bytes = parseInt(bytes);  // int64s are stringified but precision isn't critical here
```

Comment:
nit: please don't reassign method arguments.  Bind a new name, so that each variable binding always has exactly one type (and where possible, exactly one value).

(I should get ESLint up and running some time Soon™ now that we have node 4...)